internal/validations/clusterlogging/validate_clusterlogging_spec_test.go

```diff
@@ -52,31 +57,85 @@ var _ = Describe("[internal][validations] ClusterLogging", func() {
 				cl.Spec.Visualization = &logging.VisualizationSpec{}
 				cl.Spec.Curation = &logging.CurationSpec{}
 				cl.Spec.Forwarder = &logging.ForwarderSpec{}
-				err := validateClusterLoggingSpec(*cl)
+				err := validateClusterLoggingSpec(*cl, k8sClient)
 				Expect(err).To(Not(Succeed()))
 			})
 			It("should fail when collection.logs is spec'd", func() {
 				cl.Spec.Collection.Logs = &logging.LogCollectionSpec{}
-				err := validateClusterLoggingSpec(*cl)
+				err := validateClusterLoggingSpec(*cl, k8sClient)
 				Expect(err).To(Not(Succeed()))
 			})
 			It("should fail when collection.type of fluentd is spec'd", func() {
 				cl.Spec.Collection.Type = logging.LogCollectionTypeFluentd
-				err := validateClusterLoggingSpec(*cl)
+				err := validateClusterLoggingSpec(*cl, k8sClient)
 				Expect(err).To(Not(Succeed()))
 			})
 			It("should fail when collection.type is empty", func() {
 				cl.Spec.Collection.Type = ""
-				err := validateClusterLoggingSpec(*cl)
+				err := validateClusterLoggingSpec(*cl, k8sClient)
 				Expect(err).To(Not(Succeed()))
 			})
 			It("should pass when collection.type is spec'd", func() {
-				err := validateClusterLoggingSpec(*cl)
+				err := validateClusterLoggingSpec(*cl, k8sClient)
 				Expect(err).To(Succeed())
 			})
 
 		})
 
 	})
 
+	Context("#validateClusterLoggingSetUp", func() {
+		var (
+			cl             *logging.ClusterLogging
+			clf            *logging.ClusterLogForwarder
+			k8sClient      client.Client
+			otherNamespace = "other-namespace"
+			otherName      = "other-name"
+		)
+		BeforeEach(func() {
+			cl = runtime.NewClusterLogging()
+			clf = runtime.NewClusterLogForwarder()
+			k8sClient = fake.NewClientBuilder().Build()
+		})
+
+		It("should pass when CL(openshift-logging/instance) only - This is a valid LEGACY use case", func() {
+			err := validateSetup(*cl, k8sClient)
+			Expect(err).To(Succeed())
+		})
+
+		It("should pass when CL(openshift-logging/instance) & CLF(openshift-logging/instance) - This is a valid LEGACY use case", func() {
+			k8sClient = fake.NewClientBuilder().WithRuntimeObjects(clf).Build()
+			err := validateSetup(*cl, k8sClient)
+			Expect(err).To(Succeed())
+		})
+
+		It("should fail when CL(ANY_NS/instance) - This is an invalid use case", func() {
```

Comment:
We should explicitly add another test  for CL & CLF  ANY_NS/instance.  This is a valid mCLF case.  I know you have test on line 124 which should cover this but lets add one anyway